precompiles/assets/decode_helper.go

```
@@ -0,0 +1,181 @@
+	if val.Sign() <= 0 {
+		return nil, fmt.Errorf(exocmn.ErrContractInputParaOrType, index, "*big.Int", ta.args[index])
+	}
+	return val, nil
+}
+
+func (ta *TypedArgs) GetRequiredBytesPrefix(index int, length uint32) ([]byte, error) {
+	val, err := ta.GetRequiredBytes(index)
+	if err != nil {
+		return nil, err
+	}
+	if uint32(len(val)) < length {
```

golangci-lint (1.62.2)

[high] G115: integer overflow conversion int -> uint32

(gosec)